responsive-web-design-piano/css/style01.css

```diff
@@ -0,0 +1,18 @@
+/* box model reset */
+html {
+    box-sizing: border-box;
+}
+
+/* 
+    Pseudo elements also need to be reset by using the `*` universal selector.
```

Comment:
Pseudo-elements also need to be reset by using the `*` universal selector.
Pseudo-elements are special keywords that follow a selector
that let you style a specific part of the selected element.
Add a CSS rule with the selector `*, *::before, *::after` and
set `box-sizing` to `inherit`.